Makefile

```diff
@@ -0,0 +1,23 @@
+########################################################################################
+
+.PHONY = fmt all clean deps
+
+########################################################################################
+
+all: vgrepo
+
+vgrepo:
+	go build vgrepo.go
+
+deps:
+	git config --global http.https://gopkg.in.followRedirects true
```

Comment:
gopkg.in is not used in project. Try to generate actual makefile with [gomakegen](https://github.com/essentialkaos/gomakegen).